localBuildRun.sh

```diff
@@ -3,4 +3,5 @@ cp engine/parser.def build/engine/parser.def
 cp -r ../tutorials build/playbooks
 cp -r environments build
 cp -r runners build
+npm test
 node build/engine/run.js
```

Comment:
In localBuildRun.ps1 you throw an error if a test fails. This should be consistent in both files.